### PR TITLE
Profile clarification

### PR DESCRIPTION
**Pull request by cconcolato**
Trying to make it clear that out-of-profile images are valid and to clarify the limits for image resolutions, bit depth and channel numbers.

***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/pull/130.html" title="Last updated on Feb 26, 2021, 11:47 PM UTC (59a66af)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/av1-avif/130/d6630a0...59a66af.html" title="Last updated on Feb 26, 2021, 11:47 PM UTC (59a66af)">Diff</a>

### CODE REVIEW

**Comment by wantehchang** [comment]
Cyril: Thank you for writing the PR. I don't understand the profiles and levels of AV1, so I asked Adrian to review this.

**Comment by leo-barnes** [comment]
The text itself looks good to me, but I also don't know the AV1 profiles and levels well enough to review the actual numbers.

Do we also want to add a Professional AVIF profile, or should we do that in a separate PR? (Or not at all?)

**Comment by cconcolato** [comment]
let's discuss the Professional profile as part of #128

**Review by leo-barnes — APPROVED** [review]
Checked the level numbers and they look correct to me.

File: index.bs [hunk 1]
```diff
@@ -354,6 +357,12 @@ NOTE: As defined in [[!MIAF]], a file that is primarily an image sequence still
 
   If <code>'[=avis=]'</code> is declared in the [=FileTypeBox=] and a profile is declared in the [=FileTypeBox=], the profile shall also enable decoding of at least one image sequence track. The profile should allow decoding of any associated auxiliary image sequence tracks, unless it is acceptable to decode the image sequence without its auxiliary image sequence tracks.
 
+  It is possible for a file compliant to this [=AV1 Image File Format=] to not be able to declare a profile, if the corresponding AV1 encoding characteristics do not match any of the defined profiles.
```
[the remark this answers is not in the document]
Comment (cconcolato) [reply index.bs]: updated.

File: index.bs [hunk 2]
```diff
@@ -145,8 +145,9 @@ url: #biblio-av1; spec: AV1; type: dfn;
 <h2 id="general">Scope</h2>
 
 [[!AV1]] defines the syntax and semantics of an [=AV1 bitstream=]. The <dfn>AV1 Image File Format</dfn> (AVIF) defined in this document supports the storage of a subset of the syntax and semantics of an [=AV1 bitstream=] in a [[!HEIF]] file.
-The [=AV1 Image File Format=] defines multiple profiles, which restrict the allowed syntax and semantics of the [=AV1 bitstream=].
+The [=AV1 Image File Format=] defines multiple profiles, which restrict the allowed syntax and semantics of the [=AV1 bitstream=] with the goal to improve interoperability.
```
[the remark this answers is not in the document]
Comment (cconcolato) [reply index.bs]: updated

**Review by leo-barnes — APPROVED** [review]

**Pull request merged**